src/redirect.rs

```diff
@@ -2,6 +2,9 @@ use std::fmt;
 
 use ::Url;
 
+#[allow(unused_imports)]
```

Comment:
Curious what the unused imports are? Is it because they are used in the tests, but no where else? If so, you can import from inside the test function.